frontends/web/src/routes/account/send/send.tsx

```diff
@@ -38,6 +38,7 @@ import { route } from '../../../utils/route';
 import { UnsubscribeList, unsubscribe } from '../../../utils/subscriptions';
 import { ConfirmingWaitDialog } from './components/dialogs/confirm-wait-dialog';
 import { SendGuide } from './send-guide';
+import { TProposalError, txProposalErrorHandling } from './services';
```

Comment:
nit: are you planing to add more functions to services.ts, else maybe name it proposals.ts ?

Reply:
yes, there'll be more functions.